src/app/layout/header/header.component.html

```diff
@@ -114,6 +111,33 @@
   </ul>
 </div>
 
+<div id="prices_options" class="z-10 hidden bg-secondary rounded-lg shadow w-52 border border-emerald-400 border-solid">
+  <ul
+    class="py-2 px-1 text-sm text-gray-700 dark:text-gray-200"
+    aria-labelledby="dropdownDefaultButton"
+    (click)="closePrices()">
+    <li class="py-2">
+      <a class="navbar_link" routerLink="/server_prices" >
+        <lucide-icon class="h-4 w-4" name="dollar-sign"></lucide-icon>
```

Comment:
the icon shouldn't be the dollar sign here anymore, but use the one for servers

![image](https://github.com/user-attachments/assets/ad4cfaf8-5495-4112-b2b1-5e1d85c955d8)